python/tests/integration/.env

```diff
@@ -1 +1,5 @@
+LOOKERSDK_CLIENT_ID=${LOOKERSDK_CLIENT_ID}
+LOOKERSDK_CLIENT_SECRET=${LOOKERSDK_CLIENT_SECRET}
+LOOKERSDK_BASE_URL=${LOOKERSDK_BASE_URL}
+LOOKERSDK_VERIFY_SSL=${LOOKERSDK_VERIFY_SSL}
```

Comment:
these are so that if you run `pipenv run tox` and don't have a "../looker.ini" and use LOOKERSDK env vars instead, they'll actually make it through so the code under test can read them!